eng/common/pipelines/templates/steps/sparse-checkout.yml

```diff
@@ -0,0 +1,34 @@
+parameters:
+  - name: Paths
+    type: object
+    default: []
+  - name: Repositories
+    type: object
+    default:
+      - Name: $(Build.Repository.Name)
+        Commitish: $(Build.SourceVersion)
```

Comment:
LOL... what is commitish? Why not just CommitSha?

Reply:
This is an [official git term](https://git-scm.com/docs/gitglossary#Documentation/gitglossary.txt-aiddefcommit-ishacommit-ishalsocommittish), though I knew I'd raise some eyebrows :) Technically it means you can pass a commit or a **branch name**, or any other valid object ref. I'm open to a better known noun that still explains what valid values are available.

Reply:
I knew we could pass anything here that resolves to a commit but I think this is the first time I've seen this used. I'm fine keeping it.